Bindings/simulation.i

```diff
@@ -100,7 +100,6 @@ OpenSim::ModelComponentSet<OpenSim::Constraint>;
 OpenSim::ModelComponentSet<OpenSim::Controller>;
 %include <OpenSim/Simulation/Model/ControllerSet.h>
 
-%template(ModelComponentSetExternalForces) OpenSim::ModelComponentSet<OpenSim::ExternalForce>;
 %include <OpenSim/Simulation/Model/ExternalLoads.h>
```

Comment:
@aymanhab what's the reason for removing this?

Reply:
Line duplicated in the same file, hard to see in the massive list of warnings, unrelated tho to this PR